docs/89_glossary.md

```diff
@@ -126,9 +126,69 @@ following requirements:
   * Something you know (Example: passwords)
   * Something you are (Example: biometrics)
 
-* **wrong outcome** – When the reported outcome does not match the
-actual outcome. This can happen due to various reasons like
-adjudication or equipment errors.
+* **wrong outcome** - When the reported outcome does not match the
+outcome that a full hand count would show.
+This can happen due to equipment failures, adjudication errors,
+and other reasons.
+
+* **evidence-based elections* - an approach to achieving election integrity
+in which each election provides affirmative evidence that the reported
+outcomes actually reflect how people voted. This is done via
+software-independent voting systems, compliance audits and risk-limiting
+audits.  An alternative to certifying voting equipment and hoping that it
+functions properly in real elections.  See also *resilient canvass
+framework*.  See See https://www.stat.berkeley.edu/~stark/Preprints/evidenceVote12.pdf
+
+* **resilient canvass framework** - a fault-tolerant approach to conducting
+elections that gives strong evidence that the reported outcome is correct,
+or reports that the evidence is not convincing.  See also *evidence-based elections*.
+
+* **compliance audit** - an audit which checks that the audit trail is
+sufficiently complete and accurate to tell who won. Generally includes poll
+book accounting, ballot accounting, chain of custody checks, security
+checks, signature verification audits, voter registration record auditing,
+etc.  Related terms include election canvass, ballot reconciliation.
+See https://www.stat.berkeley.edu/~stark/Preprints/evidenceVote12.pdf
+
+* **audit board** - a group of electors in each county nominated by the
+major party chairpersons, which carries out an audit, with the assistance
+of the designated election official, members of his or her staff, and other
+duly appointed election judges.
+
+* **audit** TBD May include ballot tabulation audits, compliance audits, ...
+
+* **ballot tabulation audits** TBD. including risk-limiting audits,
+opportunistic audits, bayesian audits, fixed-percentage audits, etc.
+
+* **opportunistic audit* - an auditing technique designed to efficiently
+generate evidence for additional contests in a ballot-level audit. A
+significant part of the effort in doing a risk limiting audit involves
```

Comment:
Nit/question: it should generally be written "risk-limiting", not "risk limiting", right?